content/post/clojure-ast.md

```diff
@@ -0,0 +1,162 @@
+---
+author: erizocosmico
+date: 2017-02-25
+title: "The many roads towards a Clojure AST"
+draft: false
+description: "Extracting the AST of a Clojure program is super easy, you could think. It is a lisp, after all. We will be explaining our journey trying to extract the AST of Clojure programs using the `clojure.tools.analyzer` and how it was not as trivial as we thought."
+categories: ["ast", "clojure", "technical"]
+---
+
+## A little bit of context
+
+At source{d} we are starting a new ambitious project, called [bblfsh](https://github.com/bblfsh), a service to be able to parse and extract ASTs from any programming language. As a part of that project, we were developing the `clojure-driver`, which is in charge of parsing and extracting an AST from Clojure source code files.
```

Comment:
I think the project itself is called `babelfish`, but let's check with @mcuadros.